File: web-admin/src/main/java/com/isystk/sample/web/admin/controller/html/stock/regist/StockRegistForm.java
package com.isystk.sample.web.admin.controller.html.stock.regist;

import com.isystk.sample.web.base.controller.html.BaseForm;
import javax.validation.constraints.NotBlank;
import org.springframework.format.annotation.NumberFormat;

public class StockRegistForm extends BaseForm {

  private static final long serialVersionUID = 7593564324192730932L;

  @NotBlank
  String name;

  @NotBlank
  String detail;

  @NotBlank
  @NumberFormat
  String price;

  @NotBlank
  @NumberFormat
  String quantity;

  @NotBlank
  String stockImageName;

  @NotBlank
  String stockImageData;

  public @NotBlank String getName() {
    return this.name;
  }

  public @NotBlank String getDetail() {
    return this.detail;
  }

  public @NotBlank String getPrice() {
    return this.price;
  }

  public @NotBlank String getQuantity() {
    return this.quantity;
  }

  public @NotBlank String getStockImageName() {
    return this.stockImageName;
  }

  public @NotBlank String getStockImageData() {
    return this.stockImageData;
  }

  public void setName(@NotBlank String name) {
    this.name = name;
  }

  public void setDetail(@NotBlank String detail) {
    this.detail = detail;
  }

  public void setPrice(@NotBlank String price) {
    this.price = price;
  }

  public void setQuantity(@NotBlank String quantity) {
    this.quantity = quantity;
  }

  public void setStockImageName(@NotBlank String stockImageName) {
    this.stockImageName = stockImageName;
  }

  public void setStockImageData(@NotBlank String stockImageData) {
    this.stockImageData = stockImageData;
  }
}

File: web-admin/src/main/kotlin/com/isystk/sample/web/admin/controller/html/stock/regist/StockRegistForm.kt
package com.isystk.sample.web.admin.controller.html.stock.regist

import com.isystk.sample.web.base.controller.html.BaseForm
import org.springframework.format.annotation.NumberFormat
import javax.validation.constraints.NotBlank

class StockRegistForm : BaseForm() {

    @NotBlank
    var name: String? = null

    @NotBlank
    var detail: String? = null

    @NotBlank
    @NumberFormat
    var price: String? = null

    @NotBlank
    @NumberFormat
    var quantity: String? = null

    @NotBlank
    var stockImageName: String? = null

    @NotBlank
    var stockImageData: String? = null

    companion object {
        private const val serialVersionUID = 7593564324192730932L
    }
}